### PR TITLE
Requires is now 1.0

### PR DESCRIPTION
**Pull request by rofinn**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Requires = "0.5"
```
Comment (oxinabox) [inline Project.toml]: Can we allow both?

Comment (rofinn) [reply Project.toml]: I guess not. AFAICT, there's just some performance benefits in 1.0.

**Comment by rofinn** [comment]
Updated both... feel free to squash.

**Comment by oxinabox** [comment]
LGTM

**Comment by nickrobinson251** [comment]
Thanks!

**Pull request merged**